### PR TITLE
Updates Pull Request Template

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Removes a redundant line and changes to *one* relevant reviewer.

### How do I test this PR?

You don't.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set a relevant reviewer

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
I recommend to completely remove the "Questions to consider section" and add two more check boxes for the reviewer. Something like 1. Check code 2. Check test

**Comment by IsaacHardy** [comment]
What you have already changed, looks great.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,28 +13,20 @@
...
 - [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
-- [ ] Closes [DEV-XXX](#url)
 - [ ] No new warnings in tests, in storybook, and in-app
 - [ ] Upload GIF(s) of iOS and Android if applicable
-- [ ] Set two relevant reviewers
+- [ ] Set a relevant reviewer
 
 ## REVIEW
 
+- [ ] Test the fix/feature
```
Comment (IsaacHardy) [inline .github/PULL_REQUEST_TEMPLATE.md]: I meant to check unit tests, snapshots, etc...

Comment (redreceipt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Review updates to test coverage and snapshots
```

**Review by IsaacHardy — DISMISSED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ### What does this PR do, or why is it needed?
 
+### Was anything changed? Why?
```
Comment (redreceipt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+
```
Comment (redreceipt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

**Review by IsaacHardy — APPROVED** [review]

**Pull request merged**